mesonbuild/interpreterbase.py

```diff
@@ -340,17 +340,21 @@ def found_method(self, args, kwargs):
 def is_disabler(i):
     return isinstance(i, Disabler)
 
+def is_arg_disabled(arg):
+    if is_disabler(arg):
+        return True
+    if isinstance(arg, list):
+        for i in arg:
+            if is_arg_disabled(i):
+                return True
```

Comment:
I think ti would be better to return an explicit `False` at the end fo the function, and add a return annotation `bool`

Reply:
good idea, done